tracetools/CMakeLists.txt

```
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 project(tracetools)
 
-# Default to C++14
+# Default to C++17
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
```

I'm assuming you need this here too

```suggestion
  set(CMAKE_CXX_STANDARD 17)
  set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

---

Yes, of course, thank you.